packages/components/stepper/src/Stepper.doc.mdx

```diff
@@ -20,12 +21,113 @@ npm install @spark-ui/stepper
 ## Import
 
 ```tsx
-import { Stepper } from "@spark-ui/stepper"
+import { Stepper } from '@spark-ui/stepper'
 ```
 
 ## Props
 
-<ArgTypes of={Stepper} />
+<ArgTypes
+  of={Stepper}
+  description="A container for all the stepper component parts."
+  exclude={[
```

Comment:
I'm curious why we need to manually exclude this, are they not excluded by our global storybook filter by default ? (We didn't have to do this on other components)

Reply:
I agree with you, but I thought native react props were automatically omitted with our setup